Linking errors

I was using version 0.1.0 of icecream-cpp for a while, and I have `IC` statements in multiple source files, which are compiled independently (as cmake object libraries) and then linked together into different binaries and shared libraries. This used to work very nicely. Now I upgraded to 0.3.0, recompiled everything, and get linking errors like so:
```
multiple definition of `icecream::detail::show_c_string()'
```

I know that this is potentially an issue with header only libraries in general, but perhaps there is something you can do, as it used to work in previous versions.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,5 +29,5 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 include_directories(${Boost_INCLUDE_DIRS})
 
-add_executable(test_exec test.cpp)
+add_executable(test_exec test.cpp test_2.cpp)
 add_test(NAME test COMMAND test_exec)

--- a/icecream.hpp
+++ b/icecream.hpp
@@ -55,8 +55,8 @@
 
 #define ICECREAM_MAJOR_VERSION 0
 #define ICECREAM_MINOR_VERSION 3
-#define ICECREAM_PATCH_VERSION 0
-#define ICECREAM_LIB_VERSION "0.3.0"
+#define ICECREAM_PATCH_VERSION 1
+#define ICECREAM_LIB_VERSION "0.3.1"
 
 
 #if defined(__GNUC__)
@@ -1294,7 +1294,7 @@ namespace icecream{ namespace detail
         }
     };
 
-    auto show_c_string() -> bool
+    inline auto show_c_string() -> bool
     {
         return Icecream::instance().show_c_string_;
     }

--- a/test_2.cpp
+++ b/test_2.cpp
@@ -1,0 +1,3 @@
+// Header used to catch multiple symbol definition
+
+#include "icecream.hpp"

